integrations/rwightman-timm/setup_integration.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# Integration setup command to setup the folder so it is ready to train and sparsify models.
+# Creates a pytorch-image-models folder next to this script with all required dependencies from the rwightman/pytorch-image-models repository.
+# Command: `bash setup_integration.sh`
+
+git clone https://github.com/neuralmagic/pytorch-image-models.git
+cd pytorch-image-models
```

Comment:
@KSGulin we should add a release branch on the timm repo and change to that branch here so we can guarantee ongoing changes and improvements won't affect the latest stable release